crates/store/re_data_loader/src/mcap/schema/sensor_msgs/image.rs

```diff
@@ -107,10 +144,51 @@ impl McapMessageParser for ImageMessageParser {
                 .collect()
         };
 
-        let chunk = Chunk::from_auto_row_ids(ChunkId::new(), entity_path, timelines, images)
-            .map_err(|err| PluginError::Other(anyhow::anyhow!(err)))?;
+        let image_chunk = Chunk::from_auto_row_ids(
+            ChunkId::new(),
+            entity_path.clone(),
+            timelines.clone(),
+            images,
+        )
+        .map_err(|err| PluginError::Other(anyhow::anyhow!(err)))?;
+
+        let metadata_chunk = Chunk::from_auto_row_ids(
```

Comment:
Does it really make sense to have a new chunk here? It looks like the metadata is pretty small?

Reply:
The typing is a bit annoying, but let me see if I can make it work in a nicer way.